src/components/scroll-view/scroll-view.js

```diff
@@ -148,20 +154,23 @@ export default class ScrollView extends Component {
   render() {
     const {
       items,
-      length,
       itemHeight,
       scrollable,
       fullWidth,
       queryListName,
       prevNextButtons,
+      offset: page,
+      isMainPageSearch,
     } = this.props;
 
     const {
       offset,
       visibleItems,
     } = this.state;
 
-    let listHeight = (length || items.length) * itemHeight;
+    let listHeight = isMainPageSearch
```

Comment:
I think this should be a separate function. Having nested ternary operators makes this difficult to understand